docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -45,6 +45,29 @@ There you can find links to upgrade notes for other versions too.
     - from `getPaginatedProductDetailsInCategory` to `getPaginatedProductsInCategory`
     - from `getPaginatedProductDetailsForBrand` to `getPaginatedProductsForBrand`
     - from `getPaginatedProductDetailsForSearch` to `getPaginatedProductsForSearch`
+- remove usages of inherited `OrderItem` classes ([#715](https://github.com/shopsys/shopsys/pull/715))
+    - replace usages of `OrderProduct`, `OrderPayment`, and `OrderTransport` with common `OrderItem`
+    - replace usages of `OrderTransportData`, `OrderPaymentData` with common `OrderItemData`
+    - replace usages of `OrderProductFactoryInterface` with common `OrderItemFactoryInterface`
+        - replace usages of `OrderProductFactory` with `OrderItemFactory`
+        - replace usages of method `create()` with `createProduct()`
+    - replace usages of `OrderPaymentFactoryInterface` with common `OrderItemFactoryInterface`
+        - replace usages of `OrderPaymentFactory` with `OrderItemFactory`
+        - replace usages of method `create()` with `createPayment()`
+    - replace usages of `OrderTransportFactoryInterface` with common `OrderItemFactoryInterface`
+        - replace usages of `OrderTransportFactory` with `OrderItemFactory`
+        - replace usages of method `create()` with `createTransport()`
+    - replace usages of `OrderPaymentDataFactoryInterface` with common `OrderItemDataFactoryInterface`
+        - replace usages of `OrderPaymentDataFactory` with common `OrderItemDataFactory`
+        - replaces usages of method `createFromOrderPayment()` with `createFromOrderItem()`
+    - replace usages of `OrderTransportDataFactoryInterface` with common `OrderItemDataFactoryInterface`
+        - replace usages of `OrderTransportDataFactory` with common `OrderItemDataFactory`
+        - replaces usages of method `createFromOrderTransport()` with `createFromOrderItem()`
```

Comment:
I would suggest grouping all the above required steps together like this:

```
    - replace usages of `OrderProductFactoryInterface`, `OrderPaymentFactoryInterface` and `OrderTransportFactoryInterface` with common `OrderItemFactoryInterface`
        - replace usages of `OrderProductFactory`, `OrderPaymentFactory` and `OrderTransportFactory` with `OrderItemFactory`
        - replace usages of method `create()` with `createProduct()`, `createPayment()` or `createTransport()`, respectively
    - replace usages of `OrderPaymentDataFactoryInterface` and `OrderTransportDataFactoryInterface` with common `OrderItemDataFactoryInterface`
        - replace usages of `OrderPaymentDataFactory` and `OrderTransportDataFactory` with common `OrderItemDataFactory`
        - replaces usages of method `createFromOrderPayment()` and `createFromOrderTransport()` with `createFromOrderItem()`
```

It seems clearer to me and it's much shorter.

Reply:
I don't care, much... so why not :+1: